### PR TITLE
Updated README.md with running function on serverless platforms

### PR DESCRIPTION
**Pull request by meteatamel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,3 +86,14 @@ Cloud Run and Cloud Run on GKE both implement the [Knative Serving
 API](https://www.knative.dev/docs/). The Functions Framework is designed to be
 compatible with Knative environments. Just build and deploy your container to a
 Knative environment.
+
+## Configure the Functions Framework
```
Comment (jskeet) [inline README.md]: This information now occurs twice in the README - is it worth just replacing the existing occurrence with this section, perhaps?

Comment (meteatamel) [reply README.md]: Yes, I think it's nice to have a separate section for all configuration

Comment (jskeet) [reply README.md]: I'm fine with having it as a separate section - I just don't think it *also* needs to be in the "Run the function" section.

Comment (meteatamel) [reply README.md]: How about we merge "Run the function" section with the previous section and in the previous section, just show how to run it without getting into config details? This is also consistent with python and nodejs frameworks

Comment (jskeet) [reply README.md]: Yup, that's fine. I'm happy for you to do that in this PR, if you're happy to do it.

**Comment by jskeet** [comment]
Merging, thanks!

**Pull request merged**